Mathlib/Data/Polynomial/Basic.lean

```diff
@@ -1231,6 +1231,10 @@ theorem C_sub : C (a - b) = C a - C b :=
   RingHom.map_sub C a b
 #align polynomial.C_sub Polynomial.C_sub
 
+theorem monomial_sub (n : ℕ) : monomial n (a - b) = monomial n a - monomial n b := by
```

Comment:
Can you put that next to `Polynomial.monomial_neg` and also prove `Polynomial.monomial_zpow`?

Reply:
I'm not clear what `monomial_zpow` would be. There is `Polynomial.monomial_pow` already. `zpow` would presumably be for `(monomial n c)^(p : ℤ)`, but that isn't defined since polynomials don't allow for negative powers.

Such a lemma _could_ be defined for any `AddMonoidAlgebra F G` with `[Field F] [Group G]`, though?